src/web/app/components/sessions-table/sessions-table.component.ts

```diff
@@ -85,6 +85,12 @@ export class SessionsTableComponent implements OnInit {
   @Output()
   downloadSessionResultsEvent: EventEmitter<number> = new EventEmitter();
 
+  @Output()
+  sendRemindersToAllNonSubmittersEvent: EventEmitter<number> = new EventEmitter();
+
+  @Output()
+  sendRemindersToSelectedNonSubmittersEvent: EventEmitter<number> = new EventEmitter();
+
```

Comment:
Can I check if this replaces `sendRemindersToStudentsEvent` on line 80? Is the old event still necessary?

Reply:
Yup the old event is not needed, checked to make sure that there are no references to it